options.go

```diff
@@ -130,11 +130,12 @@ type Options struct {
 	// Set Verbose to true to enable more text logging.
 	Verbose bool `yaml:"verbose"`
 
-	// DEPRECATED: set `UseThrift` to true if you do not want gRPC
-	UseGRPC bool `yaml:"usegrpc"`
-
-	// Switch to
+	// Force the use of a specific transport protocol.
+	// If multiple are set to true, the top one is chosen.
+	// If none are set to true, GRPC is defaulted to.
```

Comment:
Let's be explicit about ordering thrift -> http -> grpc in the comments here since one might accidentally re-order them in the code below.

Reply:
+1